kitsune/settings.py

```diff
@@ -1042,6 +1040,15 @@ def JINJA_CONFIG():
 ]
 
 ALLOWED_HOSTS = config('ALLOWED_HOSTS', default='', cast=Csv())
+
+# Add AWS internal IP to allowed hosts for ELB health check to work.
+try:
+    EC2_PRIVATE_IP = requests.get('http://169.254.169.254/latest/meta-data/local-ipv4',
```

Comment:
We block access to the metadata server in our K8s clusters. Let's discuss options during our SUMO meeting today.